README.md

```diff
@@ -22,86 +25,28 @@ For intra-service messaging, see [Taskhawk](https://github.com/Automatic/taskhaw
 ### Installation
 
 Add to Cargo.toml:
-```
+
+```toml
 [dependencies]
-hedwig = "*"
+hedwig = "1"
 ```
 
-To use Google Publisher, use feature `google`. There's also a mock publisher provided for testing
-that can be accessed using feature `mock`.
-
-### Usage
+To use the Google Publisher, enable the `google` feature as well:
 
-```rust
-    let schema = r#"
-{
-  "$id": "https://hedwig.standard.ai/schema",
-  "$schema": "https://json-schema.org/draft-04/schema#",
-  "description": "Example Schema",
-  "schemas": {
-      "user-created": {
-          "1.*": {
-              "description": "A new user was created",
-              "type": "object",
-              "x-versions": [
-                  "1.0"
-              ],
-              "required": [
-                  "user_id"
-              ],
-              "properties": {
-                  "user_id": {
-                      "$ref": "https://hedwig.standard.ai/schema#/definitions/UserId/1.0"
-                  }
-              }
-          }
-      }
-  },
-  "definitions": {
-      "UserId": {
-          "1.0": {
-              "type": "string"
-          }
-      }
-  }
-}"#;
-
-    fn router(t: &MessageType, v: &MajorVersion) -> Option<&'static str> {
-        match (t, v) {
-            (&MessageType::UserCreated, &MajorVersion(1)) => Some("dev-user-created-v1"),
-            _ => None,
-        }
-    }
-
-    // create a publisher instance appropriately
-    // let publisher = GooglePublisher::new(String::from("/home/.google-key.json"), "myproject".into())?; 
-
-    let hedwig = Hedwig::new(
-        schema,
-        "myapp",
-        publisher
-        router,
-    )?;
-
-    let message = hedwig.message(
-        MessageType::UserCreated,
-        Version(MajorVersion(1), MinorVersion(0)),
-        UserCreatedData { user_id: "U_123" },
-    )?;
-
-    let message_id = message.id;
-
-    let publish_id = hedwig.publish(message)?;
-    println!(
-        "Published message {} with pubsub id: {}",
-        message_id, publish_id
-    );
+```toml
+[dependencies]
+hedwig = { version = "1", features = ["google"] }
 ```
 
-There's also a [full running example](examples/publish.rs) in the repo.
+### Usage
+
+See [the examples](https://github.com/standard-ai/hedwig-rust/tree/master/examples) in the
```

Comment:
unrelated to this PR - but is there an example showing sync usage as well? Or is tokio now a requirement

Reply:
Some executor, not necessarily tokio, is a requirement to drive a future to completion. It can be as simplistic as just a plain loop waiting for future to complete, though that's inefficient.

Reply:
We can write an example after a release I think.